tools/dataset_tools/glb_mesh_tools.py

```diff
@@ -0,0 +1,667 @@
+#!/usr/bin/env python
+
+# glb_mesh_tools : Tools for analyzing and modifying glb meshes
+
+import contextlib
+from collections import defaultdict
+from typing import Any, Dict, List, Optional, Set, Tuple
+
+import numpy as np
+import trimesh
```

Comment:
For the functionality we use trimesh depends on networkx, scikit-image, shapely, rtree, pyembree,